pontoon/base/management/commands/refine_translation.py

```diff
@@ -0,0 +1,37 @@
+from django.core.management.base import BaseCommand
+from pontoon.gpt.openai_connection import OpenAIService
+
+
+class Command(BaseCommand):
+    help = "Refines machine translations using OpenAI with specified characteristics"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "characteristic",
+            type=str,
+            choices=["informal", "formal", "alternative"],
+            help="The specified characteristic of the translation (informal, formal, alternative)",
+        )
+        parser.add_argument(
+            "english_text", type=str, help="The source string in English"
+        )
+        parser.add_argument(
+            "target_text", type=str, help="The machine-generated translation to refine"
+        )
+        parser.add_argument(
+            "locale_name",
```

Comment:
We're actually using language, not locale, so I'd replace all `locale*` variables with `language*`.